CONTRIBUTING.MD

```diff
@@ -4,8 +4,22 @@
 
 ## Getting started
  * Make sure you have a [GitHub account](https://github.com/signup/free)
- * Install [git-flow](https://danielkummer.github.io/git-flow-cheatsheet/#setup)
  * Clone the repository
 
 ## Encountered a bug ?
 If it doesn't already exists, create a new [issue](https://github.com/OSAlt/nya-chan-bot/issues), and make sure you give enough details for us to reproduce the bug.
+
+## Want to help resolving an issue ?
+ * Check Nya Chan's [Project](https://github.com/OSAlt/nya-chan-bot/projects) to make sure the issue is not already being taken care of.
+ * Create a new branch and do your magic.
```

Comment:
You don't need this since everyone is forking the project. This only applies to people with write access which will be a handful